frontend/src/features/mapSideBar/SidebarContents/subforms/AssociatedLandReviewPage.tsx

```diff
@@ -110,6 +114,23 @@ export const AssociatedLandReviewPage: React.FC<any> = (props: IReviewProps) =>
       usage: editInfo.usage && isParcelValid,
       valuation: editInfo.valuation && isParcelValid,
     };
+    const currentYear = moment().year();
+    const assessedIndex = indexOfFinancial(
```

Comment:
Need to get the desired indexes for all of these values in the mixed evaluations and fiscals arrays.